controllers/comments.controller.js

```diff
@@ -9,4 +9,15 @@ exports.getCommentsForArticle = (req, res, next) => {
     .catch(err => {
         next(err)
     })
+}
+
+exports.addNewComment = (req, res, next) => {
+    const articleId = req.params.article_id
+    const username = req.body.username
+    const comment = req.body.body
+    return commentsModel.addCommentToArticleForUser(comment, articleId, username)
+    .then( comment => {
+        res.status(201).send({ comment })
+    })
+    .catch(err => next(err))
```

Comment:
You could experiment with destructuring as a more streamlined syntax if you felt comfortable doing so:
e.g.

const {article_id} = req.params

or one step further

const { params: { article_id }} = req